src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+    private static final Lottery lottery1 = new Lottery();
```

Comment:
Why do you need this object as a field? When you have created another object inside main() (which is the correct approach)

Reply:
word object helps me. fixed